docs/src/rsh/errors/index.md

```diff
@@ -1373,6 +1373,10 @@ In `{!rsh} parallelReduce` or `{!rsh} fork`, a `{!rsh} timeout` is required unle
 always races, the `{!rsh} when` field in their `PUBLISH_EXPR` is statically `{!rsh} true`, or if one
 `{!rsh} ParticipantClass` will attempt to race.
```

Comment:
Add `API` to statement:
`{!rsh} ParticipantClass` or `{!rsh} API` will attempt to race.